### ISSUE TITLE
Register D-Bus Names

### ISSUE THREAD

**Issue by landersson** [LinkTed/dbus-async-derive#3]
When running the "methods" example, I can call Method using for example:

dbus-send --session --print-reply --dest=:1.170 /org/example/methods org.example.methods.Method

Where :1.170 happens to be the unique name of the service. The example doesn't seem to register a name for the service.

I'd like to be able to specify a name, to call it using for example:

dbus-send --session --print-reply --dest=org.example.methods /org/example/methods org.example.methods.Method

Is this currently possible?

**Comment by LinkTed** [comment]
Yes, it is currently supported. See [`DBus`](https://docs.rs/dbus-async/2.0.0/dbus_async/struct.DBus.html#method.register_name)
